### PR TITLE
Install libmediainfo-config and libmediainfo.pc

### PR DESCRIPTION
**Pull request by g-maxime**
Signed-off-by: Maxime Gervais <gervais.maxime@gmail.com>

### CODE REVIEW

**Comment by JeromeMartinez** [comment]
/debian and .spec should be updated at the same time in order to avoid duplicated code.

**Comment by g-maxime** [comment]
Done

+ Removed sed commands (version already set by ./configure script)

**Comment by JeromeMartinez** [comment]
> Removed sed commands (version already set by ./configure script)

Great (having it centralized is better), but I still see one "sed". Expected?

File: Project/GNU/PKGBUILD [hunk 1]
```diff
@@ -29,8 +29,6 @@ package() {
     install -dm 755 $pkgdir/usr/include/$i
     install -m 644 $srcdir/MediaInfoLib/Source/$i/*.h $pkgdir/usr/include/$i
   done
-  install -dm 755 $pkgdir/usr/lib/pkgconfig
-  install -m 644 $srcdir/MediaInfoLib/Project/GNU/Library/libmediainfo.pc $pkgdir/usr/lib/pkgconfig
   sed -i -e 's|Version: |Version: '$pkgver'|g' $pkgdir/usr/lib/pkgconfig/libmediainfo.pc
```
Comment (g-maxime) [inline Project/GNU/PKGBUILD]: Oups, ;)

And same for ZenLib:
```
Name: libzen
Version: 0.4.330.4.33
```

**Pull request merged**